resources/lang/de/base.php

```diff
@@ -105,12 +105,12 @@
                     'title' => 'Server Informationen',
                 ],
                 'suspend' => [
-                    'description' => 'Ermöglicht das Suspendieren einer Serverinstanz.',
+                    'description' => 'Ermöglicht das suspendieren einer Serverinstanz.',
                     'title' => 'Server suspendieren',
                 ],
                 'title' => 'Serververwaltung',
                 'unsuspend' => [
-                    'description' => 'Ermöglicht die Suspendierung einer Serverinstanz aufzuheben.',
+                    'description' => 'Ermöglicht die suspendierung einer Serverinstanz aufzuheben.',
```

Comment:
Why you changed that?

Reply:
Woops, good to see, i think i was reading too fast that line.. 

I'll change it back. 
Thank you for your help